stock_picking_security/security/stock_security.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="group_stock_picking_partner" model="res.groups">
+            <field name="name">Partners</field>
+            <field name="category_id" ref="base.module_category_operations_inventory"/>
+            <field name="implied_ids" eval="[(4, ref('stock.group_stock_user'))]"/>
+        </record>
+
+        <record id="stock.menu_stock_inventory_control" model="ir.ui.menu">
+            <field name="groups_id" eval="[(4,ref('stock.group_stock_manager'))]"/>
+        </record>
+
+        <record id="stock.menu_action_inventory_form" model="ir.ui.menu">
+            <field name="groups_id" eval="[(4,ref('stock.group_stock_manager'))]"/>
+        </record>
+
+        <record id="stock.menu_stock_scrap" model="ir.ui.menu">
+            <field name="groups_id" eval="[(4,ref('stock.group_stock_manager'))]"/>
+        </record>
+
+        <record id="stock.menu_procurement_compute" model="ir.ui.menu">
+            <field name="groups_id" eval="[(4,ref('stock.group_stock_manager'))]"/>
+        </record>
+
+        <record id="stock.group_stock_manager" model="res.groups">
+            <field name="implied_ids" eval="[(6, 0, [ref('stock_picking_security.group_stock_picking_partner'), ref('account.group_account_invoice')])]"/>
```

Comment:
Tried to install, faced an error:
```
"External ID not found in the system: account.group_account_invoice" while evaluating
"[(6, 0, [ref('stock_picking_security.group_stock_picking_partner'), ref('account.group_account_invoice')])]
```
And that's right, making this addon depending on `stock`, `account` addon shouldn't be used.

You'd get the desired behavior changing this (maybe you should test it and check if `account` permissions are kept):

```suggestion
            <field name="implied_ids" eval="[(4,ref('stock_picking_security.group_stock_picking_partner'))]"/>
```